pipelined/bevy_gltf2/src/loader.rs

```diff
@@ -27,7 +27,7 @@ use gltf::{
     Material, Primitive,
 };
 use std::{
-    collections::{HashMap, HashSet},
+    collections::{HashMap, HashSet, VecDeque},
```

Comment:
Might as well use `bevy_utils::Hash{Map|Set}`

Reply:
Indeed. I intended to do so, but the dependency relation wasn't pre-existing. 

Thanks for the reminder!